server/src/test/java/io/crate/integrationtests/CopyIntegrationTest.java

```diff
@@ -554,18 +554,17 @@ public void testCopyToWithWhereNoMatch() throws Exception {
 
     @Test
     public void testCopyFromNestedArrayRow() throws Exception {
-        // assert that rows with nested arrays aren't imported
         execute("create table users (id int, " +
             "name string) with (number_of_replicas=0)");
         execute("copy users from ? with (shared=true)", new Object[]{
             nestedArrayCopyFilePath + "nested_array_copy_from.json"});
-        assertEquals(1L, response.rowCount()); // only 1 document got inserted
+        assertEquals(2L, response.rowCount());
         refresh();
 
         execute("select * from users");
-        assertThat(response.rowCount(), is(1L));
+        assertThat(response.rowCount()).isEqualTo(2L);
 
-        assertThat(printedTable(response.rows()), is("2| Trillian\n"));
+        assertThat(response).hasRows("1| Ford", "2| Trillian");
```

Comment:
maybe was some raw specific old limitation  - in CSV we just ignore extra columns if target is not specified, I think in json we can also ignore extra columns (maybe add only if table is dynamic)